platforms/windows/hax_wrapper.c

```diff
@@ -237,47 +226,70 @@ void hax_disable_irq(void)
     asm_disable_irq();
 }
 
-void hax_error(char *fmt, ...)
+int hax_log_level(int level, const char *fmt,  ...)
```

Comment:
int hax_log_level(int level, const char *fmt, va_list arglist)
{
const char *level_prefix[] = {
"",
"haxm_debug:",    // HAX_LOGD
"haxm_info:",        // HAX_LOGI
"haxm_warning:", // HAX_LOGW
"haxm_error:"       // HAX_LOGE
};
if (level >= HAX_LOG_DEFAULT)
vDbgPrintExWithPrefix(level_prefix[level], DPFLTR_IHVDRIVER_ID, DPFLTR_ERROR_LEVEL, fmt, arglist);
return 0;
}

Reply:
Considering define prefix as below to prevent this array being created and destroyed over and over again:
static const char *prefix[] = {
"",
"haxm_debug: ", // HAX_LOGD
"haxm_info: ", // HAX_LOGI
"haxm_warning: ", // HAX_LOGW
"haxm_error: " // HAX_LOGE
};

Reply:
I have refine the log accordingly.